language/nodejs/run-containers.md

```diff
@@ -162,7 +162,7 @@ ec45285c456d        node-docker         "docker-entrypoint.s…"   31 minutes ag
 fb7a41809e5d        node-docker         "docker-entrypoint.s…"   40 minutes ago      Exited (0) 39 minutes ago                            goofy_khayyam
 ```
 
-To remove a container, simple run the `docker rm` command passing the container name. You can pass multiple container names to the command in one command.
+To remove a container, simply run the `docker rm` command passing the container name. You can pass multiple container names to the command in one command.
```

Comment:
We should probably remove `simply` altogether (it's good to avoid words like `simply` or `simple` or `easy`, because what's easy for the "writer" may not be "easy" to the reader; especially in a getting started section 😅).